configuration/driver/icedrv_step.F90

```diff
@@ -717,6 +798,7 @@ subroutine step_dyn_ridge (dt, ndtd)
       use icedrv_state, only: trcrn, vsnon, aicen, vicen
       use icedrv_state, only: aice, aice0, trcr_depend, n_trcr_strata
       use icedrv_state, only: trcr_base, nt_strata
+      use icedrv_state, only: divu
```

Comment:
is divu used?

Reply:
No. I've removed the line. Here's a long-winded explanation of what I was thinking about:

I had originally included divu here because divu (and shear for that matter) are always equal to zero. I thought that icepack_step_ridge was using the rdg_conv and rdg_shear values to calculate opening and closing. Then I would update the divu value as `divu(i) = opening(i) - closing(i)` after the call to icepack_step_ridge. In standalone icepack runs that's not the case because we simply read in the opening and closing. Also, if the ridging step needs to iterate (as it does anytime we have closing and `ice_advection_type = "none"`) opening and closing get overwritten anyways. We could compute divu from opening and closing before running the ridging step, but that could also be computed directly from the forcing data. So in the current configuration, I'm not sure it makes sense to include divu and shear in the history output at all. I guess it saves a user from having to query the forcing files in post-processing.

Also, I'm chatting with the ice dynamics experts from MOSAiC to try to determine how to relate the observations we've made to an appropriate dynamics forcing for single column models. I'm not sure yet if our resulting product will be opening and closing rates or something else (e.g., like observations of the net advection and amount of ridging convergence and shear). So my plan is to revisit the whole divu question after we sort out the form of the dynamics forcing.